src/App.css

```diff
@@ -1,6 +1,65 @@
-.auth-container{
-    background-image: url('./assets/images/bg3.png');
-    background-position:0ex;
-    height: 100vh;
+/* ---------------------- common style ---------------------- */
 
+.main-colr {
+    color: rgba(239, 155, 40, 1);
+}
+
+.main-bg {
+    background-color: rgba(239, 155, 40, 1);
+}
+
+.sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
+    white-space: nowrap;
+    border-width: 0;
+}
+
+/* ---------------------- auth-layout ----------------------*/
+.auth-container {
+    background-image: url("./assets/images/bg3.png");
+    background-position: center center;
+    background-size: cover;
+
+    label {
+        font-size: 0.8rem;
+    }
+
+    input {
```

Comment:
We shouldn't make these styles global, as inputs in the application  have different styles. Making them global will introduce css classes conflicts.
We need to make them inside a css module file to be scoped to the file they are used in.

Reply:
The styles have been handled and made local within the component using CSS nesting, preventing class conflicts with the rest of the application.

Reply:
What I meant here is to change it to fileName.modules.css